File: 380/a.rb
def cyclic(n)
  a = n.to_s.chars
  b = a.rotate
  c = b.rotate
  "#{b.join} #{c.join}"
end

def main(input)
  lines = input.split("\n")
  cyclic(lines[0])
end


if __FILE__ == $PROGRAM_NAME
  input = ARGF.read
  lines = input.split("\n")
  puts main(lines[0])
end

# require "minitest/autorun"

# describe 'test' do
#   it 'test' do
#     _(cyclic(379)).must_equal '793 937'
#     _(cyclic(919)).must_equal '199 991'
#   end
# end

File: 380/a.py
# python -m doctest -v a.py
from pprint import pp as pp

def cyclic(n):
  '''
  >>> cyclic('379')
  '793 937'

  >>> cyclic('919')
  '199 991'
  '''
  a = list(n)
  b = a[1:] + a[0:1]
  c = b[1:] + b[0:1]
  return f'{''.join(b)} {''.join(c)}'


def main():
    import sys
    lines = sys.stdin.read().split()
    pp(lines)
    ret = cyclic(lines[0])
    print(ret)


if __name__ == "__main__":
    main()
